src/Setup/WixBA/InstallationViewModel.cs

```diff
@@ -254,6 +277,33 @@ public bool UninstallEnabled
             get { return this.UninstallCommand.CanExecute(this); }
         }
 
+        private ICommand openLogCommand;
```

Comment:
Agreed.  I new I put it there and when I went back to make that change I
missed it.

On Thu, Jul 28, 2016 at 3:04 AM, Rob Mensching <notifications@github.com>
wrote:

> In src/Setup/WixBA/InstallationViewModel.cs
> <https://github.com/wixtoolset/wix4/pull/208#discussion_r72579723>:
>
> > @@ -254,6 +277,33 @@ public bool UninstallEnabled
> >              get { return this.UninstallCommand.CanExecute(this); }
> >          }
> >
> > +        private ICommand openLogCommand;
>
> Style: We stick all privates at the top of the class.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/wixtoolset/wix4/pull/208/files/5b08dba511d7438508767effab67896442e4ad1b#r72579723>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHyCLUkk7Bi6zvlPyOGuB99HouPFbrUCks5qaGJ9gaJpZM4JVm8P>
> .
>